### ISSUE TITLE
Trojan:Win32/Emali.A!cl 

### ISSUE THREAD

**Issue by dorofino** [grayfallstown/Chia-Plot-Status#50]
I'm getting a warning about a Trojan Emali.A!cl  on the Avalonia.Themes.Default.dll

Is anyone else getting this warning?

![image](https://user-images.githubusercontent.com/1240729/118560880-2fa00580-b738-11eb-8192-92f8f55463f4.png)

**Comment by PiSCaRGoT** [comment]
Yup, I'm getting it too. Was about to post myself.

**Comment by dorofino** [comment]
I'm not sure if this is a false positive, but for now, I uninstalled it.

Trojan:Win32/Emali.A!cl
Detected by Microsoft Defender Antivirus
Aliases: No associated aliases

Summary
Microsoft Defender Antivirus detects and removes this threat.

**This threat can perform a number of actions of a malicious hacker's choice on your PC.**

**Comment by rochonno** [comment]
I just got this too. Glad I am not alone. This is probably a false positive. It is a fairly common occurrence to have windows be over-protective. I have worked on deep learning software that gets blocked by windows due to "unusually high activity for extended time periods". 

I know some developers will submit there installers to Microsoft for verification, although I highly doubt there is a quick turnaround on that.

**Comment by grayfallstown** [comment]
duplicate of https://github.com/grayfallstown/Chia-Plot-Status/issues/47#issuecomment-842418163
File is from an external library and is currently processed by malware analysts.

**Comment by grayfallstown** [comment]
As you can read in #47 updating Chia Plot Status to its newest version solves the issue (but while installing you might still get a hit on the old files beeing replaced)

You can read more on the avalonia repo: https://github.com/AvaloniaUI/Avalonia/issues/5937

**Comment by grayfallstown** [comment]
```
avalonia.win32.dll
Submission ID: 2e5978b5-3b0c-414b-9e30-33e2407c3194
Status: Completed
Submitted: May 17, 2021 17:24:14
User Opinion: PuaFalsePositive
Analyst comments:

We have removed the detection. Please follow the steps below to clear cached detection and obtain the latest malware definitions.

1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender
2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
3. Run "MpCmdRun.exe -SignatureUpdate"

Alternatively, the latest definition is available for download here: https://www.microsoft.com/en-us/wdsi/definitions

Thank you for contacting Microsoft.
```

**Comment by duckduckwho** [comment]
Now it shows Trojan:Win32/Sabsik.FT.A!ml

**Comment by grayfallstown** [comment]
@duckduckwho with which version?

**Comment by duckduckwho** [comment]
https://github.com/grayfallstown/Chia-Plot-Status/releases/latest/download/Setup.exe

**Comment by grayfallstown** [comment]
Submitted Avalonia.Dialogs.dll for analysis too.

**Comment by duckduckwho** [comment]
"
1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender
2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
3. Run "MpCmdRun.exe -SignatureUpdate""

Now it works.

**Comment by dorofino** [comment]
Great, it looks like is all clear now.
No Trojan detected. 

I thought that it was a false positive anyway.
Thank you for clearing this up. 

Btw, this is an excellent tool.
Thank you.

**Comment by grayfallstown** [comment]
I am closing this now, but I leave the notice informing about this incident in the readme for a while.
